src/caf/distribute/gravity_model/multi_area.py

```diff
@@ -563,29 +676,35 @@ def _gravity_function(
         return achieved_residuals
 
     # pylint:enable=too-many-locals
-    def run(self):
+    def run(
+        self,
+        distributions: list[MultiCostDistribution],
```

Comment:
This shouldn't be a list anymore
```suggestion
        distributions: MultiCostDistribution,
```